fast64_internal/f3d/f3d_bleed.py

```diff
@@ -438,6 +437,12 @@ def bleed_individual_cmd(self, cmd_list: GfxList, cmd: GbiMacro, bleed_state: in
         if not last_cmd_list:
             return self.bleed_self_conflict
 
+        # Don´t bleed if the cmd tags are not the same (these are not hashed)
+        if cmd in last_cmd_list:
```

Comment:
I think it would be better if this behavior was done inside the cmd specific bleed logic. There is no reason it couldn't be done there since really this only affects set tile size cmds.
In the case that tags have some general use to every type of gfx cmd it would make sense to have it here, but if it is for set tile size put it in the set tile size function

Reply:
I was thinking of future proofing I guess, but if that happens it would be better to just hash the tags